css-canvas-medical/styles.css

```diff
@@ -0,0 +1,192 @@
+html {
+  font-family: Avenir, sans-serif;
+  font-size: 14pt;
+}
+
+.margin {
+  margin-left: 4rem;
+  margin-right: 4rem;
+}
+
+.heading-font {
+  font-family: 'Tiempos Headline', serif;
+  font-size: 2rem;
+}
+
+.center-text {
+  text-align: center;
+}
+
+.header-container {
+  display: flex;
+  max-width: min(1600px, calc(100% - 8rem));
+  margin: 1.5rem auto 1.5rem auto;
+}
+
+.bottom-border {
+  margin-bottom: 3rem;
+  border-bottom: 2px solid rgb(231, 233, 236);
+}
+
+.header-section {
+  margin-top: auto;
+  margin-bottom: auto;
+  width: 50%;
+}
+
+.header-image {
+  max-height: 1.5rem;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  color: rgb(160, 160, 160);
+}
+
+.contact-navbar {
+  padding: 0.5rem 1.25rem 0.5rem 1.25rem;
+  color: white;
+  background-color: rgb(41, 68, 100);
+  border-radius: 5px;
+}
+
+.homepage-content1 {
+  color: white;
+  background-image: url(images/hero.jpg);
+  height: 800px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-attachment: initial;
+  max-width: min(1600px, calc(100% - 8rem));
+  margin: auto auto 3rem auto;
+  position: relative;
+}
+
+.overlay {
+  background: rgba(90, 90, 90, 0.33);
+  background-position: center;
+  width: min(100%, 1600px);
+  height: 100%;
+  margin: auto;
+}
+
+.homepage-content1 > section {
+  position: absolute;
+  top: 3rem;
+  left: 4rem;
+  line-height: 2rem;
+}
```

Comment:
the content of the text should be arranged using margin and padding , it is not a valid use for absolute positioning